### PR TITLE
Enabled lint check "ObsoleteLayoutParam"

### LINKED ISSUE #10482: Enable lint `ObsoleteLayoutParam`
We perform lint checking to ensure code correctness and improve performance, but many of the system lint checks have not been enabled.

The goal of this issue is to enable the lint check: `ObsoleteLayoutParam`

Line: https://github.com/ankidroid/Anki-Android/blob/6260cd6e25f67f331833bbd0981f2cda5a426dde/lint-release.xml#L266

To do this: 

* Change `ignore` to `fatal`
* Move the line in the XML to be somewhere near the top of the file (before the large block of `ignore` elements. 
  * The exact position is up to you, but if there's no comment associated with the check, a large nearly alphabetical block of `fatal` warnings seems sensible
* Run `gradlew lint` in the Android Studio terminal and fix the errors which occur.
  * `@Suppress(str)` (Kotlin) or `@SuppressWarnings(str)`  (Java) the warnings if they are valid. Android Studio's quickfixes menu will offer options to suppress
    * Use your judgment, but aim to suppress as close to the error as possible (line/method) unless the class specifically has a reason why it will violate a lint check many times
  * If it was an error that we missed, fix it.
* Submit a Pull Request

### PR DESCRIPTION
**Pull request by anamansari062**
## Purpose / Description
Enabled lint check for "ObsoleteLayouParam", changed "ignore" to "fatal" and took the line above.

## Fixes
Fixes  #10482 

## How Has This Been Tested?

Ran ./gradlew lint and resolved errros in activity_drawing.xml

### CODE REVIEW

**Comment by anamansari062** [comment]
@david-allison Please Check.

File: AnkiDroid/src/main/res/layout/activity_drawing.xml [hunk 1]
```diff
@@ -9,20 +9,16 @@
...
         layout="@layout/reviewer_whiteboard_editor"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_centerHorizontal="true"
-        android:layout_above="@+id/bottom_area_layout"/>
+        android:layout_height="wrap_content" />
     <FrameLayout
         android:layout_gravity="center"
         android:id="@+id/whiteboard"
-        android:layout_below="@id/whiteboard_editor"
```
Comment (Akshay0701) [inline AnkiDroid/src/main/res/layout/activity_drawing.xml:25]: @anamansari062 you changed the alignment of UI here, any specific reason for these changes?

**Review by Akshay0701 — CHANGES_REQUESTED** [review]
@anamansari062 Thanks for your contributions!

I just have one question for changes you made on the drawing XML file, the rest looks good.

**Comment by anamansari062** [comment]
@Akshay0701 @david-allison  do i need to any more changes?

File: AnkiDroid/src/main/res/layout/activity_drawing.xml (hunk 1 above)
Comment (Akshay0701) [reply AnkiDroid/src/main/res/layout/activity_drawing.xml:25]: revert this changes

**Review by david-allison — CHANGES_REQUESTED** [review]
Did you run `gradlew lint`? It should have listed changes to make.

**Comment by anamansari062** [comment]
@david-allison 
image
It requested the same changes i did in earlier commit, but reverted in the new one.

**Comment by anamansari062** [comment]
@david-allison Can you tell me the steps to go to drawing activity, I am new to the code base thatswhy.

**Comment by anamansari062** [comment]
# Ran gradlew lint and got no errors
Screenshot 2022-03-15 at 12 01 22 AM

# Also the alignment is proper in drawing activity

image

**Comment by anamansari062** [comment]
@david-allison Can i commit the changes now?

**Review by david-allison — APPROVED** [review]
As long as this has been tested in an emulator, I'm happy. Thanks!!!!

**Review by mikehardy — APPROVED** [review]
Very nice! Thanks for this @anamansari062

**Comment by mikehardy** [comment]
Hi there! MARCH OpenCollective Notice

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for [March 2022 submissions](https://github.com/ankidroid/Anki-Android/pulls?q=is%3Apr+merged%3A2022-03-01..2022-03-31)

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged for in the month of March.

Please note that GSoC contributions are okay for this process. Our philosophy is that our users have donated to AnkiDroid for *all* contributions. The only PRs that will not go through the OpenCollective process are ones *directly* related to a GSoC project from a selected participant, since those receive a stipend from GSoC itself.

Please understand that this means the monthly budget we have for payments might be stretched a bit and the cap on payments-per-person may be lower, but we try to make our process as fair and transparent as possible, we just need your understanding.

Thanks!

**Pull request merged**